### PR TITLE
fix the demos so they compile for the internal brand

### PR DESCRIPTION
**Pull request by JakeChampion**
🤦 
currently they don't compile because of the use of the `paper` color for the background

### CODE REVIEW

File: demos/src/demo.scss [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+		background-color: oColorsByName('paper');
+		margin: 1rem;
+		padding: 1rem;
+	}
+}
+@if oBrandGetCurrentBrand() == 'internal' {
+	.demo {
+		margin: 1rem;
+		padding: 1rem;
+	}
```
Comment (JakeChampion) [inline demos/src/demo.scss:15]:
```suggestion
	.demo {
		background-color: oColorsByUsecase('page', 'background')
		margin: 1rem;
		padding: 1rem;
	
```

File: demos/src/demo.scss [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-.demo {
-	background-color: oColorsByName('paper');
-	margin: 1rem;
-	padding: 1rem;
+	.demo {
+		background-color: oColorsByUsecase('page', 'background')
+		margin: 1rem;
+		padding: 1rem;
+	
 }
```
Comment (JakeChampion) [inline demos/src/demo.scss:9]:
```suggestion
.demo {
	background-color: oColorsByUsecase('page', 'background')
	margin: 1rem;
	padding: 1rem;
}
```

**Review by notlee — APPROVED** [review]

**Comment by JakeChampion** [comment]
Not quite sure why the tests on the npm version are failing, it is because the swc bundler can't find ftdomdelegate.

When I run the swc bundle command myself though it works 😕 

This works for me and fails in CI
```
cat src/js/share.js |  npx swc --filename - --no-swcrc -C minify --source-maps inline
```

**Comment by notlee** [comment]
@JakeChampion seems odd, what happened between the previous PR where the build passed and now 😕

**Comment by JakeChampion** [comment]
looks like the latest release of origami-build tools is the issue

the previous version passes the tests
```
❯ npx origami-build-tools@10.8.5 test --ignore-bower
Downloading Chromium r686378 - 110.2 Mb [====================] 100% 0.0s 
 ✔ Testing sass compilation
   ✔ Testing dart-sass compilation for the internal brand with silent mode off
 ↓ running true test [skipped]
   → No sass test files found in ./test/scss
 ↓ Executing `npm test` [skipped]
 ✔ Executing Pa11y
 ✔ Running Karma tests on Chrome Stable
   ✔ Completed tests on Chrome Headless 86.0.4240.198 (Mac OS 10.15.7) in 0.226 seconds. Ran 14 tests. 14 tests successful. 0 tests failed. 
```

the latest version fails the tests
```
❯ npx origami-build-tools@10.8.6 test --ignore-bower
Downloading Chromium r686378 - 110.2 Mb [====================] 100% 0.0s 
 ❯ Testing sass compilation
   ⠼ Testing node-sass compilation with silent mode on
 ↓ running true test [skipped]
   → No sass test files found in ./test/scss
 ↓ Executing `npm test` [skipped]
 ✖ Executing Pa11y
   → Failed building Sass:
     ' Error: Can't find stylesheet to import.
       ╷
     1 │ @import 'o-brand/main';
       │         ^^^^^^^^^^^^^^
       ╵
       ../../main.scss 1:9  @import
       - 2:9                root stylesheet
     /Users/jake.champion/Code/main.scss:1:9
 ❯ Running Karma tests on Chrome Stable
   ⠼ Starting Karma server
```

**Comment by JakeChampion** [comment]
The new version of obt changed how the pa11y test is ran and I forgot to ensure the new way worked for the npm version of our components -- pr here to fix it -- https://github.com/Financial-Times/origami-build-tools/pull/934

**Comment by origamiserviceuser** [comment]
o-share bundle size difference from 7.5.0 to 7b280211323505cbdb6d4eeb18250f9c40ef802a
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v7.5.1 :tada:

**Pull request merged**